operator/Makefile

```diff
@@ -32,6 +34,9 @@ endif
 SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
+# Compute the current Git commit hash
+HASH=$(shell git rev-parse HEAD)
```

Comment:
this would not work if we want to keep it empty on purpose. When the Dockerfile invokes the build, if `HASH` was empty on purpose it will try to compute again the Git hash but it will fail since there is no Git repo in the Docker context